docs/chapter1/dicts/clean-code/src/example.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+type Point struct {
+	x int
+	y int
+}
+
+var points = []Point{{4, 5}, {7, 1}, {2, 9}}
+
+const maxFloat64 = 1.79769313486231570814527423731704356798070e+308
```

Comment:
マジックナンバーの説明としてこの書き方を入れたいのはわかるのだけれど、実際は`math.MaxFloat64`を呼び出すような書き方をすべきなのでそれを軽く触れられるとよさそう

本文で触れるのを避けたければ、こういったコメント行でソースを示すだけでも良いとは思います
```suggestion
// math.MaxFloat64の値
// https://pkg.go.dev/math#pkg-constants
const maxFloat64 = 1.79769313486231570814527423731704356798070e+308
```

Reply:
ここでGoの正しい書き方を教えたいわけじゃないので書かなくていいかなと思ってます

コメントとかもこの章で扱っている以上余計な情報で混乱するのを避けたい

Reply:
疑似コードなら何でもいいんですが、Go余計な機能がなくてちょうどいいのでやってるだけです（C++とちょっと迷った）